### PR TITLE
fix: Update validation for Daily Round form

### LINKED ISSUE #1775: Update validation for Daily Round form
**Describe the bug**
Currently there are lots of validations in daily rounds FE that are not there in BE.

Such as, validation to prevent empty log updates from being created.

- [ ] Add proper serializer validations for daily rounds.

### PR DESCRIPTION
**Pull request by aeswibon**
# Bug Fix

## Proposed Changes

- [x] Fixes #1775 

## Merge Checklist

- [x] Tests added/fixed
- [ ] Update docs in `/docs`
- [x] Linting Complete
- [ ] Any other necessary step

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@coronasafe/care-backend-maintainers @coronasafe/care-backend-admins

### CODE REVIEW

**Comment by aeswibon** [comment]
@sainak Can you check the PR?

**Comment by sainak** [comment]
@aeswibon missing validation for other fields

File: care/facility/api/serializers/daily_round.py [hunk 1]
```diff
@@ -331,6 +342,23 @@ def validate(self, attrs):
...
+        if "additional_symptoms" in validated:
+            if SymptomChoiceDict["OTHERS"] in validated["additional_symptoms"]:
+                if "other_symptoms" not in validated:
+                    raise ValidationError(
+                        {
+                            "other_symptoms": [
+                                "This field is required if OTHERS is selected in Additional Symptoms."
+                            ]
+                        }
+                    )
```
[the remark this answers is not in the document]
Comment (aeswibon) [reply care/facility/api/serializers/daily_round.py]: Done

**Comment by aeswibon** [comment]
@sainak I have added the validation

**Pull request closed without merging**